src/main/java/org/sbml/spatial/segmentation/TrainUnetModel.java

```diff
@@ -117,14 +116,15 @@ public static void main(String[] args) {
 
 			int numEpochs = 1;
 
-			ComputationGraph model = UNet.builder().updater(new Adam(1e-4)).build().init();
+			// ComputationGraph model = UNet.builder().updater(new
+			// Adam(1e-4)).build().init();
 			// ComputationGraph model = UNet.builder().updater(new Adam(new
 			// MapSchedule(ScheduleType.ITERATION, learningScheduleMap))).build().init();
 
 			// To change the number of input channels from 3 to 1
-			// ZooModel unet = UNet.builder().build();
-			// unet.setInputShape(new int[][]{{1, 128, 128}});
-			// ComputationGraph model = (ComputationGraph) unet.init();
+			ZooModel unet = UNet.builder().build();
```

Comment:
I think it's best to set the optimizer to Adam, just as you did with 3 channels.
```
ZooModel unet = UNet.builder().updater(new Adam(1e-4)).build();
```

Reply:
Will make the necessary changes. Thanks.